aerogearcatalog/aerogear-digger-apb
# update path poc

# Digger update path

## Changes

**Parameters**
- ANDROID_SDK_AUTO_CONFIG: boolean blag to install the default packages defined in digger apb
- ANDROID_SDK_CONFIG: custom config string to be used when installing packages (ignored if `ANDROID_SDK_AUTO_CONFIG` is set to true)

**Playbooks**

- A new default.cfg file to be used when ANDROID_SDK_AUTO_CONFIG is set to true
- New update playbook and role

**Verification steps:**

- Provision digger apb with the "ANDROID_SDK_AUTO_CONFIG" field checked - it should deploy the default packages described in the default.cfg file
- Uncheck "ANDROID_SDK_AUTO_CONFIG" and add the following config string in "ANDROID_SDK_CONFIG": https://github.com/aerogear/androidctl/blob/master/examples/android.cfg
- Provision the apb without both vars and update it by adding the above config config string into the  `ANDROID_SDK_CONFIG` field
